### PR TITLE
fix: increase feature column gaps

### PR DESCRIPTION
**Pull request by christian-dinh**
### Overview

I took out the `xs` gaps because they never actually got applied as far as I could tell (it would switch to the single-column view at the same time). Dev says this is "perfect" 🙃 

### PR Checklist

- [ ] Related to designs:
- [x] Related to JIRA ticket: [REACH-602](https://codecademy.atlassian.net/browse/REACH-602)
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

File: packages/gamut-labs/src/landingPage/PageFeatures.tsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ const rowRenderEach = (
       ))}
     </FlexContainer>
   );
-  /* eslint-ensable react/no-array-index-key */
+  /* eslint-enable react/no-array-index-key */
```
Comment (rogervera) [inline packages/gamut-labs/src/landingPage/PageFeatures.tsx:56]: Hehe

**Review by rogervera — APPROVED** [review]
Nice. Playing around a bit with `xl` which is `3rem`, looks good on our careers page.

Screen Shot 2021-02-08 at 5 33 02 PM

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@9.2.8-alpha.bf5775.0<br/>@codecademy/styleguide@25.1.9-alpha.bf5775.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6021c2bc9b69db319ed26c73--gamut-preview.netlify.app>https://6021c2bc9b69db319ed26c73--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6021c2bc9b69db319ed26c73>Deploy Logs</a></p>

**Pull request merged**